OWNERS

```diff
@@ -0,0 +1,21 @@
+approvers:
+  - knobunc
+  - Miciah
+  - frobware
+  - danehans
+  - sgreene570
+  - candita
+  - rfredette
+  - alebedev87
+  - miheer
+reviewers:
+  - knobunc
+  - Miciah
+  - frobware
+  - danehans
+  - sgreene570
+  - candita
+  - rfredette
+  - alebedev87
+  - miheer
+component: DNS
```

Comment:
We should think about adding a new Bugzilla component for ExternalDNS (and likewise for Contour).

Reply:
Noted. We can worry about that once we are at a point where we are filing bugzilla reports :)